pipelines/smode_iop1_saildrone/test/test_pipeline.py

```diff
@@ -0,0 +1,16 @@
+import xarray as xr
+from pathlib import Path
+from utils.a2e_tsdat import PipelineConfig, assert_close
+
+# Commented out the test due to the output file's large size, which makes it too difficult to upload.
```

Comment:
You should be able to save a much smaller subset of the datafile to use for this test.